config_example.sh

```diff
@@ -822,6 +822,11 @@ set -x
 #
 # export APPLIANCE_IMAGE="quay.io/edge-infrastructure/openshift-appliance:latest"
 
+# Sets which extraworker node to add to cluster using the agent add-node workflow.
```

Comment:
I think it will be helpful to have a wrteup/help showing what are the steps required before running node-joiner. Per my understanding, 
1. Build the openshift-binary
2. Use it to generate agent ISO as per your config
3. Boot up the ISO and Create a cluster
4. Now, if you want to add extra nodes, do the following
5. Build the node-joiner binary by doing xyz
6. Use this new binary as described below to add nodes 
     1. details
     2. additional details 
     3. etc